airbyte-integrations/connectors/source-amazon-ads/acceptance-test-config.yml

```diff
@@ -14,12 +14,19 @@ tests:
   basic_read:
     - config_path: "secrets/config.json"
       configured_catalog_path: "integration_tests/configured_catalog.json"
-      empty_streams: ["sponsored_brands_campaigns", "sponsored_brands_ad_groups", "sponsored_brands_keywords"]
-      expect_records:
-        path: "integration_tests/expected_records.txt"
-        extra_fields: no
-        exact_order: no
-        extra_records: no
+      empty_streams:
+        [
+          "sponsored_brands_ad_groups",
+          "sponsored_brands_campaigns",
+          "sponsored_brands_keywords",
+          "sponsored_product_keywords",
+          "sponsored_product_negative_keywords",
+          "sponsored_product_targetings",
```

Comment:
Why this streams: sponsored_product_keywords, sponsored_product_negative_keywords, sponsored_product_targetings

added as empty ?

Reply:
it comes from me, irrelevant to the topic branch. It was about 2 weeks ago, the integration test says these streams are empty in the test account when I run it locally. And I have the login, but I can't access the OTP token, so I can't login.